app/ai/voice/agents/breeze_buddy/template/global_function.py

```
@@ -27,6 +30,121 @@
+
+    template = getattr(bot_instance, "template", None)
+    music_config = (
+        template.configurations.function_call_music
+        if template and template.configurations
+        else None
+    )
+    if music_config and music_config.enabled:
+        context = TemplateContext(bot_instance)
+        try:
+            await context.set_mixer_enabled(False)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)